### ISSUE TITLE
Endless login loop

### ISSUE THREAD

**Issue by hhinma** [warpdotdev/Warp#1780]
### Describe the bug

Having downloaded and installed warp (unable to determine version since the about box is greyed out),   application asks to signup.  Email received and link clicked on.   From this point,  running the app produces a login dialog box, and clicking on any of the UI controls results in a new browser window with the same login dialog box.     Endless loop.  No obvious way out of this.   Deleting and reinstalling the app,  the same behavior occurs, without even the option to sign up.   Essentially non-functional product. 

MacBook M1
MacOS Monterery 12.0.1

### To Reproduce

Repeat steps above

### Expected behaviour

Some kind of rational login behavior, where signing up according to the instruction results in being able to access the application.

### Operating System

MacOS

### OS Version

12.0.1

### Shell Version

N/A no shell accessible

### Warp Version

Setting menu not accessible,  neither is the about box.

### Does this block you from using Warp daily?

Yes

**Comment by dannyneira** [comment]
@hhinma Just to confirm, are you using any VPN or Proxy that may be blocking the login url?

**Comment by alokedesai** [comment]
Hey @hhinma, thanks for flagging! Could you record a video of the issue occurring? Do any of the other sign up methods work?

**Comment by alokedesai** [comment]
@hhinma Friendly ping on this! We would love to help debug this and understand what's going on here. Thanks so much!

**Comment by hhinma** [comment]
The problem is now resolved - i dont have any clue why unfortunately 

harveyharrison.net

**Comment by warpdotdev-dx** [comment]
Thanks for letting us know!

**Comment by codersk** [comment]
Hi All, I am facing the same issue. Any clue or workaround?

**Comment by alokedesai** [comment]
@codersk Could you elaborate more on the exact issue you are experiencing? Are you able to login successfully if you try to sign up via google or github directly?

**Comment by hhinma** [comment]
It appears you did not read the update on September 3rd.   This is now resolved.

**Comment by codersk** [comment]
@alokedesai I signed up using GItHub. But when I try to sign in I see "You‘re logged in as Sandeep" screen on browser but app stuck at login screen. I also tried to copy paste the Auth Token method, but no luck.
![Screenshot 2022-10-18 at 3 42 26 PM](https://user-images.githubusercontent.com/8155010/196402714-8dfe4a52-4c55-4c9f-9f5c-81cccf07c00b.png)

**Comment by phubbard** [comment]
I'm having the same problem - new corporate machine, M1 based 14". Let me see if dropping VPN resolves it.

**Comment by phubbard** [comment]
Nope, same problem. Monterey 12.6.1, new M1 Max, Chrome browser, trying to use my github login but Warp won't get past the 'Sign up / Login' banner. No errors in console. Same result using Safari.

**Comment by jackbow** [comment]
I'm having the same issue. I tried the advice from https://docs.warp.dev/help/troubleshooting-login-issues to no avail. M1 Max mac with macos v13.0.1 on Chrome. On Firefox I get a screen that says "Error!" after logging in and trying and failing to open Warp. On Safari I get "Oops! We were unable to sign you in. Please try again" when trying to login via email.

**Comment by zachbai** [comment]
Thanks all for the reports here. I'm taking over on this issue from @alokedesai.

@phubbard: 
> Nope, same problem. Monterey 12.6.1, new M1 Max, Chrome browser, trying to use my github login but Warp won't get past the 'Sign up / Login' banner. No errors in console. Same result using Safari.
Just confirming - you're able to sign in with GitHub and redirected back to app.warp.dev, but nothing happens in the native app? 

@jackbow If either you're still able to reproduce, can you relay the contents from your browser log console from any of the browsers where you're encountering the issue? You can also email me directly at zachbai[at]warp[dot]dev.

**Comment by zachbai** [comment]
Thanks @jackbow. And confirming that 'pasting' the refresh token while the native app is focused does nothing?

I'm suspecting that the app's Firebase request to fetch an access token is failing, and for whatever reason similar Firebase auth requests succeed in your browser -- hence why you have a valid refresh token but nothing happens in the app. Do you have any browser extensions installed that might affect network traffic? This could be an adblocker, VPN, corporate proxy etc.

Last Wednesday, we released a version of Warp that logs network requests and responses -- if you downloaded Warp since then, you should have a log file at `~/Library/Application Support/dev.warp.Warp-Stable/warp_network.log` with entries for each request and response.  Are there any response items with error statuses? Look for a `Response` entry with `host` = `securetoken.googleapis.com` and `path` = `v1/token`.

**Comment by zachbai** [comment]
Thanks for bearing with me here, the log sample is helpful. Just to confirm -- there are no corresponding `Response` entries for these requests?

**Comment by jackbow** [comment]
Embarrassingly I forgot I have [lulu](https://objective-see.org/products/lulu.html) installed and warp got on its block list without my knowledge. I removed the entry and its working fine now. Thanks for looking into it with me. Might be wise to add a error stating warp can't connect to the internet and to check firewalls so someone like me doesn't come to bother y'all again.

**Comment by zachbai** [comment]
@jackbow No worries and thanks for following up! 

Like you suggested, I'm adding an error notification to the signup screen for such situations when the network request fails.
